examples/heroku/README.md

```diff
@@ -0,0 +1,53 @@
+# Heroky deployment graphql-faker
+
+We are going to create a package and deploy it to Heroku.
+
+# Create your sdl schema
+
+The graphql `SDL` schema is the basis of graphql-faker. If you have the schema in `JSON`, you will need to convert it to `SDL`. We've added the utility to do that right here.
+
+## JSON to SDL
+
+```
+yarn convert-graphql-json
```

Comment:
It's better to do `npx convert-graphql-json` that way it's always the latest version and we don't need to constantly update `yarn.lock` for every security vulnerability that GitHub detects.

Reply:
good point